app/scripts/modules/core/src/pipeline/manualExecution/manualPipelineExecution.html

```diff
@@ -80,7 +80,25 @@ <h3 ng-if="!vm.command.pipeline">Select Pipeline</h3>
             <span ng-if="parameter.required">*</span>
             <help-field content="{{parameter.description}}" ng-if="parameter.description"></help-field>
           </div>
-          <div class="col-md-6" ng-if="!parameter.hasOptions">
+          <div class="col-md-6" ng-if="!parameter.hasOptions && parameter.constraint === 'date'">
+            <p class="input-group">
+              <input type="text"
+                class="form-control input-sm"
+                uib-datepicker-popup
```

Comment:
Cool, so this already existed and we just never used it?

Reply:
Yeah, a part of the angular-ui-bootstrap